assets/dev/js/editor/regions/navigator/components/item-indicator-list.js

```diff
@@ -0,0 +1,30 @@
+import ItemIndicator from './item-indicator';
+import PropTypes from 'prop-types';
+
+export function ItemIndicatorList( { settings, onActivateSection } ) {
+	/**
+	 * Generate a list of indicator elements by checking whether the element contains settings that are suppose to
+	 * result in an indicator on the navigator.
+	 *
+	 * @type {ItemIndicator[]}
+	 */
+	const list = Object.entries( elementor.navigator.region.indicators ).map(
```

Comment:
It's not connected to a slice, so, one changes in the indicators, it will not update the component.

Reply:
`elementor.navigator.region.indicators` is a constant that defines the available indicators and their settings (custom css, position, etc.). It's not keeping data regarding which elements have which indicators.